src/components/Login/LoginForm.js

```diff
@@ -1,79 +1,101 @@
 import React from 'react';
-import App from '../../pages/App';
+import { useSelector, useDispatch } from 'react-redux';
+import { renderApp } from "../../rStore/actions/index";
 import logo from './logo.png';
-import UsernamePassword from './UsernamePassword';
-import validateInput from '../../rStore/validate/login';
+import { Field, reduxForm } from 'redux-form';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
 
-
-class LoginForm extends React.Component{
-
-    constructor(props) {
-        super(props);
-        this.state = {
-            username: '',
-            password: '',
-            errors: {},
-            submitted: false
-        };
-        this.onSubmit = this.onSubmit.bind(this);
-        this.onChange = this.onChange.bind(this);
-    }
-
-    isValid() {
-        const { errors, isValid } = validateInput(this.state);
-
-        if(!isValid) { 
-            this.setState({ errors });
+const validate = values => {
+    const errors = {}
+    const requiredFields = [
+        'username',
+        'password'
+    ]
+    requiredFields.forEach(field => {
+        if (!values[field]) {
+          errors[field] = 'Required'
         }
-        return isValid;
-    }
+    })
+    return errors
+} 
 
-    onSubmit(e) {
-        e.preventDefault();
-        if(this.isValid()) {
-            this.setState({ errors: {}, submitted: true });
-            // after validating the user we will call login function
-            // from rstore actions to authenticate in the db
-        }
-        console.log(this.state);
-    }
+function handleChange(e) {
+    this.setState({ [e.target.name]: e.target.value });
+}
```

Comment:
Just dont forget to make a note to convert this to. to a hook instead of using this.setState